pkg/client/client.go

```diff
@@ -11,15 +11,19 @@ import (
 	"github.com/aep-dev/aep-lib-go/pkg/api"
 )
 
+type LoggingFunction func(ctx context.Context, message string)
```

Comment:
maybe consider adding varargs, so that it can also be used if your logging interface is a structure logger (e.g. slog https://pkg.go.dev/log/slog#Logger.Info)?

Reply:
Great idea!